### PR TITLE
NFDIV-3996 Update NOP template references in application.yaml

### PR DESCRIPTION
**Pull request by pallavijustice**
### Change description ###

Update NOP template references in application.yaml

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-3996

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -379,18 +379,18 @@ docmosis:
...
-      NFD_NOP_RS1_SOLE_APP2_SOL_ONLINE: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Sole-Respondent-V3.docx
-      NFD_NOP_RS2_SOLE_APP2_SOL_OFFLINE: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Sole-Respondent-V3.docx
+      NFD_NOP_AL2_SOLE_APP1_CIT_PS: FL-NFD-GOR-ENG-Notice-Of-Proceedings-AL2-V4.docx
+      NFD_NOP_R1_SOLE_APP2_CIT_ONLINE: FL-NFD-GOR-ENG-Notice-Of-Proceedings-R1-V9.docx
+      NFD_NOP_R2_SOLE_APP2_CIT_OFFLINE: FL-NFD-GOR-ENG-Notice-Of-Proceedings-R2-V10.docx
+      NFD_NOP_R2_SOLE_APP2_CIT_OFFLINE_REISSUE: FL-NFD-GOR-ENG-Notice_Of_Proceedings_Respondent_ReIssue_Offline_V2.docx
+      NFD_NOP_R2_SOLE_APP2_OUTSIDE_ENGLAND_WALES: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Outside-England-Wales-R2-V4.docx
+      NFD_NOP_RS1_SOLE_APP2_SOL_ONLINE: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Sole-Respondent-V4.docx
+      NFD_NOP_RS2_SOLE_APP2_SOL_OFFLINE: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Sole-Respondent-V4.docx
       NFD_NOP_JA1_JOINT_APP1APP2_CIT: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Joint-V3.docx
```
Comment (adamg-hmcts) [inline src/main/resources/application.yaml:391]: Do you know if we need to update these CIT templates as well?

Comment (pallavijustice) [reply src/main/resources/application.yaml:391]: Are you referring to NFD_NOP_JA1_JOINT_APP1APP2_CIT: FL-NFD-GOR-ENG-Notice-Of-Proceedings-Joint-V3.docx?

Comment (adamg-hmcts) [reply src/main/resources/application.yaml:391]: Yep the template I've commented under. There is also `NFD_NOP_A2_SOLE_APP1_CIT_PS` on line 383

Comment (pallavijustice) [reply src/main/resources/application.yaml:391]: The ticket asked to update those templates which already had paragraph around re-issue. This template on line 391 doesn't have re-issue content.

Also NFD_NOP_A2_SOLE_APP1_CIT_PS isn't used in the code so wasn't considered. The templates I picked was based on following the code in reissueApplicationservice. There might be more templates that we currently do not use.

**Review by adamg-hmcts — APPROVED** [review]

**Pull request merged**